src/WebJobs.Script/Workers/Rpc/IWebHostRpcWorkerChannelManager.cs

```diff
@@ -11,7 +11,7 @@ public interface IWebHostRpcWorkerChannelManager
     {
         Task<IRpcWorkerChannel> InitializeChannelAsync(IEnumerable<RpcWorkerConfig> workerConfigs, string language);
 
-        Dictionary<string, TaskCompletionSource<IRpcWorkerChannel>> GetChannels(string language);
+        IDictionary<string, TaskCompletionSource<IRpcWorkerChannel>> GetChannels(string language);
```

Comment:
note this interface change... while public, it's not something we ship elsewhere...